backstage/app-config.yaml

```diff
@@ -113,3 +98,38 @@ catalog:
             minutes: 60
           timeout:
             minutes: 30
+    ldapOrg:
+      default:
+        target: ldap://ralfed.cclrc.ac.uk
+        bind:
+          dn: uid=ldap-reader-user,ou=people,ou=example,dc=example,dc=net
+          secret: dW5hdXRoZW50aWNhdGVkIExEQVAK
+        schedule:
+          frequency: PT1H
+          timeout: PT10M
+        users:
+          dn: ou=dls,dc=fed,dc=cclrc,dc=ac,dc=uk
+          options:
+            filter: (memberOf=cn=DLSLTD_Staff,OU=Automatic Groups,DC=fed,DC=cclrc,DC=ac,DC=uk)
+            paged: true
+          map:
+            name: cn
+            displayName: displayName
+        groups:
+          dn: dc=fed,dc=cclrc,dc=ac,dc=uk
+          options:
+            filter: (objectClass=group)
+
+search:
+  collators:
+    catalog:
+      schedule:
+        # Give the backend server a chance to initiaize before collators are executed
```

Comment:
This feels unnecessary :stuck_out_tongue:

Reply:
The timeout or the comment about the timeout? It was in search.tsx before.

Reply:
The comment on the timeout :laughing: Wasn't a particularly serious review